### PR TITLE
Style & layout clean up

### PR DESCRIPTION
**Pull request by magnolia61**
NOTE: this is a follow up of https://github.com/civicrm/civicrm-core/pull/16569 because git and I could not quite come to terms with each other on that one. 

Overview
----------------------------------------
The Contribution Invoice workflow template needed some quick fixes:
- style wise clean up
- some serious layout fixes
- make CiviCRM logo comply with _Display "empowered by CiviCRM"_ setting

Before 
----------------------------------------
Bit of a messy Invoice Layout
Screenshot from 2020-02-17 22-41-59

After
----------------------------------------
Clean professional layout
WITHOUT CIVICRM LOGO
Screenshot from 2020-02-17 22-43-14
WITH CIVICRM LOGO
Screenshot from 2020-02-17 22-52-02

Technical Details
----------------------------------------
I will first let the template file be tested before I add the upgrader files.
Note: the CREDIT NOTE part has not been updated yet as the template contains double code for that. When this is merged I will change it so the same layout can be used forboth invoices and credit notes.

Comments
----------------------------------------
Ideally the Contribution Invoice would use a configurable logo. I have been thinking to use the Profile picture from the system user, but I will open a gitlab to discuss the best option for this.

TODO
----------------------------------------
list of things to change in the next step
- change logic so credit note can use the same layout as the invoice
- use contribution receive date instead of current date (gitlab will follow)
- use different wording for invoice and receipt depending on contribution status
- improve wording and functionality of the payment advice (gitlab will follow)
- use configurable organisation logo for top logo. Move 'powered by' to bottom

### CODE REVIEW

**Comment by magnolia61** [comment]
Some additional screenshots:
INVOICE WITHOUT TAX - WITH PAY LATER ADVICE
Screenshot from 2020-03-01 23-10-30
INVOICE WITH TAX - WITH PAY LATER ADVICE
Screenshot from 2020-03-01 23-10-52
INVOICE WITHOUT TAX - PARTIALLY PAID
Screenshot from 2020-03-01 23-11-15

**Comment by magnolia61** [comment]
@sunilpawar mentioned this list of enhancements to the template he worked on, of which some should be discussed in gitlab, but good to mention here for a good overview:

1 .fixing empty line if address part is missing for contact, adjust space, comma between the 2 word if variable value is missing.
2. add domain_postal_code_suffix
3. Add country Abbreviation for billing contact
4. Correcting Billing Name When Payment made using on behalf of org contact.
5. Option to Expose Payment Method (Payment Instrument) for received payments.
6. Email Invoice with CC and Subject field Option.
7. Change Header 'Invoice' / 'Receipt' on Payment Status.

**Comment by sunilpawar** [comment]
@magnolia61 Here is zip file that contain invoice tpl changes, changed tpl file (you can update invoice template for testing using this file) and PDF file default and new format after applying patch.
[invoice_changes.zip](https://github.com/civicrm/civicrm-core/files/4287117/invoice_changes.zip)

**Comment by eileenmcnaughton** [comment]
I think this as is was discussed & approved in the previous Pr - I'm going to set to MOP to get it merged before the rc is cut

**Comment by eileenmcnaughton** [comment]
Unrelated fail

**Comment by magnolia61** [comment]
> I think this as is was discussed & approved in the previous Pr - I'm going to set to MOP to get it merged before the rc is cut

Thank you Eileen

File: xml/templates/message_templates/contribution_invoice_receipt_html.tpl [hunk 1]
```diff
@@ -105,246 +92,228 @@
...
-                <td colspan = "3"></td>
-                <td style = "padding-left:20px;text-align:right;"><b><font size = "1">{ts 1=$defaultCurrency}TOTAL %1{/ts}</font></b></td>
-                <td style = "padding-left:34px;text-align:right;"><font size = "1">{$amount|crmMoney:$currency}</font></td>
-                <td style = "padding-left:34px;"><font size = "1" align = "right"></font></td>
+                <td colspan="3"></td>
+                <td style="text-align:right;white-space: nowrap"><b><font size="1">{ts 1=$defaultCurrency}TOTAL %1{/ts}</font></b></td>
+                <td style="text-align:right;"><font size="1">{$amount|crmMoney:$currency}</font></td>
               </tr>
-              {if $is_pay_later == 0}
+             {if $amountDue != 0}
```
Comment (eileenmcnaughton) [inline xml/templates/message_templates/contribution_invoice_receipt_html.tpl:129]: This caused a regression - see https://github.com/civicrm/civicrm-core/pull/18602

**Pull request merged**